modules/ecs/ipfs/main.tf

```diff
@@ -43,7 +43,7 @@ resource "aws_ecs_service" "main" {
   }
 }
 
-resource "aws_ecs_task_definition" "main" {
+resource "aws_ecs_task_definition" "td" {
```

Comment:
Are people gonna have issues with this resource name change? Not sure we have a way to test this but given the issues we saw yesterday seems fine to keep it as "main" (this is just terraform convention).

Reply:
I don't think they should but I don't mind changing it back. TF will delete the old TD and create two new ones.

The problem is that it is no longer the only TD in there, so we should call them `main` and `main_existing_peer`? That looks weird to me but I don't have a strong opinion.

Or, I can leave the first one as `main` and call the new one `existing_peer`, or something like that.

Reply:
Ok, if we don't anticipate issues then all good.

Reply:
Shouldn't be any issues but I reverted to `main` to minimize changes when someone applies the module.